### ISSUE TITLE
Dataset Request

### ISSUE THREAD

**Issue by seunghub** [ghayem/NeuroConText#1]
Hello,

Thank you for your inspiring work!

Can you share the available dataset link?

Current link (https://zenodo.org/uploads/12684666) seems not working.

Thank you for your time, and I look forward to your response.

**Comment by ghayem** [comment]
Hello,

Thanks for your interest in NeuroConText and for reporting the issue with
downloading the dataset.

The link has been modified in the NeuroConText repo. You can download it
from this link:
 data_NeuroConText_MICCAI24
<https://zenodo.org/records/14169410?token=eyJhbGciOiJIUzUxMiJ9.eyJpZCI6ImM0NmUwMWZhLWVmYzQtNDUxZS05NTg3LWJjZDdhZGY5MGRiYyIsImRhdGEiOnt9LCJyYW5kb20iOiI3MDlhYjYwYWYwN2Q1Y2JmYWU0MjE0NTFlNGYzMTQxZiJ9.p7EhGnpNIBN73FOn-L5MmQ9Dz5Cx86Y9x7kZWUyVz_fTp_lLxEEb21c4aBC-wb9Fbyg7dF8r1uHycu2I_dZBXw>

Do not hesitate to contact me for any further questions.

Best,
Fateme Ghayem

**Comment by seunghub** [comment]
Thank you for sharing your dataset!

I have another question regarding reproducibility. 
Using the hyperparameters reported in the paper:
'''
batch_size = 128
lr = 5e-4
weight_decay = 0.1
dropout = 0.5
num_epochs = 50
'''
I obtained about 82.1 on the mix&match metric. 
Are the 'preprocessed_train/test_text_embeddings' derived from the Mistral-7B model? 
If so, is there anything else I need to adjust in the code to reach the reported 84.2 score?

Again, thank you for your time and assistance!

**Comment by ghayem** [comment]
Hello,

The score of 84.2 represents the average result from a 15-fold cross-validation on the test set selection. To achieve this result, mix the test and training data first. Then, randomly select 1K samples as the test set and 19K as the training set. Run the code for each of these selections. Repeat this process 15 times and calculate the average of the outcomes.

Feel free to let me know if you have any more questions.

Best,
Fateme Ghayem

**Comment by seunghub** [comment]
Thank you so much for your detailed and kind responses to all my questions. 
I really appreciate your time and effort. I’m closing the issue now.
